### PR TITLE
[MM-47964] - Update limits in Cloud Pricing Modal

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Update copy about limits in cloud pricing modal based on https://docs.google.com/document/d/1i9iuyyV-PyuhIw19gxnCEbFfe6IE2fnWOoMThiz926o/edit#

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-47964

#### Screenshots
New copy
-----
Screenshot 2023-01-10 at 11 31 01

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/pricing_modal/content.tsx [hunk 1]
```diff
@@ -151,7 +151,7 @@ function Content(props: ContentProps) {
     const hasLimits = hasSomeLimits(limits);
 
     const starterBriefing = [
-        formatMessage({id: 'pricing_modal.briefing.free.recentMessageBoards', defaultMessage: 'Access to {messages} most recent messages, {boards} most recent board cards'}, {messages: formatNumber(fallbackStarterLimits.messages.history), boards: fallbackStarterLimits.boards.cards}),
+        formatMessage({id: 'pricing_modal.briefing.free.recentMessageBoards', defaultMessage: 'Access to {messages} most recent messages, No board limits'}, {messages: formatNumber(fallbackStarterLimits.messages.history)}),
```
Comment (neallred) [inline components/pricing_modal/content.tsx:154]: This is the text in the description of the ticket, but I think the text there is to inform us internally of what the change is. I think the desired user-facing wording is in the google doc, and is to have `Unlimited board cards` as its own bullet point (https://docs.google.com/document/d/1i9iuyyV-PyuhIw19gxnCEbFfe6IE2fnWOoMThiz926o)

File: utils/limits.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const fallbackStarterLimits = {
         history: 10000,
     },
     files: {
-        totalStorage: FileSizes.Gigabyte * 10,
```
Comment (neallred) [inline utils/limits.tsx:31]: :100:

**Review by neallred — APPROVED** [review]
LGTM, thanks @AGMETEOR !

**Comment by yasserfaraazkhan** [comment]
@AGMETEOR , 
on the spin wick the text change is not visible, can you once review if I am missing anything.

Screenshot 2023-01-09 at 10 59 06 PM

**Comment by yasserfaraazkhan** [comment]
@AGMETEOR ,
few text changes are not visible on spinwick, can you review if I am missing anything

Screenshot 2023-01-09 at 10 59 06 PM

**Comment by johndavidlugtu** [comment]
@yasserfaraazkhan the screenshot you have looks to be self-hosted. This PR is for Cloud only -- @AGMETEOR correct me if I'm wrong

**Comment by johndavidlugtu** [comment]
@AGMETEOR let's use the copy from here: https://docs.google.com/document/d/1i9iuyyV-PyuhIw19gxnCEbFfe6IE2fnWOoMThiz926o/edit#heading=h.3meqjer8hdri

**Comment by johndavidlugtu** [comment]
@AGMETEOR apart from the Free plan, correct -- the main difference lies in the "250 GB file storage limit" in Cloud Professional. That's no longer the case.

In any case, use the copy in [GDoc](https://docs.google.com/document/d/1i9iuyyV-PyuhIw19gxnCEbFfe6IE2fnWOoMThiz926o/edit#heading=h.3meqjer8hdri) as a guide

**Review by yasserfaraazkhan — APPROVED** [review]
text changes are visible Allan, thank you

**Review by marianunez — APPROVED** [review]
Thanks @AGMETEOR!

**Comment by johndavidlugtu** [comment]
@AGMETEOR one last thing and we're good to go
Screenshot 2023-01-10 at 2 18 41 PM

Should be

Screenshot 2023-01-10 at 2 20 46 PM

**Review by johndavidlugtu — APPROVED** [review]

**Pull request merged**